cypress-shared/integration/location-availability/2.1-disable-location-and-test.spec.js

```diff
@@ -0,0 +1,25 @@
+import {
+  preserveCookie,
+  updateRetry,
+  loginViaAPI,
+} from '../../support/common/support'
+import selectors from '../../support/common/selectors'
+
+const prefix = 'Disable location'
+
+describe('Location validation', () => {
```

Comment:
Update this titles to meaningful name
It is same on multiple files that confuses the user

Reply:
ok